### DISCUSSION TITLE
Instagram permissions getting rejected

### DISCUSSION THREAD

**Discussion by choira-music** [chatwoot/chatwoot#6393] [Self hosted]
Hello I'm requesting an approval for **instagram_basic** and **instagram_manage_messages**

I followed this guide https://github.com/chatwoot/chatwoot/discussions/3397#discussioncomment-1911551
and recorded a screen-cast accordingly mentioned my use case but it is getting rejected.

**Notes from your reviewer:**
Hello team, at present the screencast is missing the demonstration of Story Mentions. Please take note that the contents of the Story Mention must be visible on the custom inbox app. However, story replies are not required.

Kindly ensure that the screencast and submission notes must both demonstrate and mention the following steps:

1) A Instagram user must initiate a new message to the Business Instagram Account.
2) Message must be received and replied via the custom inbox app.
3) Show that the Instagram user received the reply on the Instagram app.
4) Instagram user must mention the Business Instagram Account in their story. Then the story mention in the custom inbox app (story content must be visible or opened in a new tab).
5) Show un-sending message by Instagram user from the Instagram app.
6) Show that the message was removed from the custom inbox app too.
7) Show the use of CDN privacy URL to host media (image/story).

**instagram_basic feedback**

App rejected - Disallowed Use Case Details
We have determined that your app's use case for the requested permission is invalid or is not needed to support its core functionality.
Your app should only request for permissions that help enhance and build a solid user experience for your business and the end user. Please resolve this issue by removing or revising your app content for compliance before submitting for review.
If you believe your use case is valid, please consider submitting a revised screencast or notes that explains the following items:
1. Which functionality of your app requires the permission.
2. How will the permission to enhance your app's functionality and how the integration functions.
3. How will the permission enhance end user's experience.

**instagram_manage_messages feedback**
App rejected - App missing story mention support

During our review, we found that your screen recording and submission notes didn't fulfill the technical requirements for use of the Story mention. Your submission should show how users will respond to Story @mentions, including how media is delivered in the app.
To resolve this issue, make sure that your submission clearly shows how the Story @mention feature will function in your app.

But chatwoot doesn't support's delete message and story mention feature in instagram
please let me know which approach should i follow to get instagram running for production

**Comment by muhsin-k** [comment]
We've shipped a standalone Instagram channel, making it easier for users to connect and manage their accounts. You can read more about this [here](https://www.chatwoot.com/hc/user-guide/articles/1744361165-how-to-setup-an-instagram-channel-via-instagram-login). We've also added a template you can use as a reference for obtaining the required Instagram permissions from Meta https://www.chatwoot.com/docs/self-hosted/instagram-app-review. We're currently working on the messenger template review guide and will share it soon.
